operatorcourier/validate.py

```diff
@@ -390,6 +390,9 @@ def get_alm_kinds(alm_examples):
             for example in alm_examples:
                 res.append(example["kind"])
             return res
+        
+        def is_mediatype(mediatype):
+            return mediatype in ["image/gif", "image/jpeg", "image/png", "image/svg+xml"]
```

Comment:
Is this an exhaustive list of our supported image types?

Reply:
as far as I know yes